docs/getting_started.md

```diff
@@ -877,8 +878,6 @@ Generate an ECDSA key:
     mbedtls_psa_crypto_free();
 ```
 
-### More about the Mbed Crypto library
-
-More information on [Mbed Crypto](https://github.com/ARMmbed/mbed-crypto/).
+### More about the Mbed Crypto API
 
-More information on [PSA Crypto](https://github.com/ARMmbed/mbed-crypto/blob/development/docs/PSA_Crypto_API_Overview.pdf).
+For more information about PSA Crypto, download the *PSA Cryptography API* PDF under [PSA APIs](https://developer.arm.com/architectures/security-architectures/platform-security-architecture#implement).
```

Comment:
This URL is the official PDF which is not up-to-date. It would be better to link to the copy that's more useful to developers, which has moved to https://github.com/ARMmbed/mbed-crypto/raw/psa-crypto-api/docs/PSA_Cryptography_API_Specification.pdf and also has an HTML version at https://armmbed.github.io/mbed-crypto/html/index.html